packages/terra-embedded-content-consumer/tests/nightwatch/consumers/MessagingConsumer.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
This really should just be a jest test to ensure XFC is passed back on mount. We have thorough tests around XFC messaging in that project.